components/ImaSdkTask.xml

```diff
@@ -40,6 +40,12 @@
         <!-- Set when user requests that the video stream be cancelled. Usually by pressing back at the Choice Card. -->
         <field id="userCancelStream" type="boolean"/>
 
+        <!-- Set when user is currently snapped back to an ad from trying to seek past it. -->
+        <field id="inSnapback" type="boolean" value="false"/>
```

Comment:
Do we need these fields outside of the ImaSdkTask?

Reply:
They're not used yet but `ContentFlow` may need access to disable/enable video flags (`enableUi` for example).